Add traci calls for a vehicle type's maximum speed

heinovski opened the pull request:
I tested the new methods with the integrated veins_testsims (SUMO 0.30.0, 0.32.0, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.3.1) as well as some custom simulation I am currently working on. I used OMNET 5.4.1.  It does not change anything in the API.

dbuse commented:
Looks good! Haven't tested/run it myself yet, though.

sommer commented on subprojects/veins_testsims/src/veins_testsims/traci/TraCITestApp.cc:
```diff
@@ -205,6 +205,16 @@ void TraCITestApp::handlePositionUpdate()
+        if (t == 1) {
+            assertEqual("(TraCICommandInterface::getVehicleTypeMaxSpeed speed is correct)", traci->getVehicleTypeMaxSpeed("vtype0"), 70);
+            traci->setVehicleTypeMaxSpeed("vtype0", 60);
+            assertEqual("(TraCICommandInterface::getVehicleTypeMaxSpeed) changed speed is correct", traci->getVehicleTypeMaxSpeed("vtype0"), 60);
```
this seems to test `set...` rather than `get...`, so the comment should be updated I guess

heinovski commented:
> without also updating `omnetpp.ini` line `103`, what is now test 79 won't be executed any more.

I adjusted the omnetpp.ini. Furthermore, I changed the test according to your other comments.
